package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@manuscripts/api",
-  "version": "1.7.11-LEAN-3321-0",
+  "version": "1.7.11-LEAN-3321-1",
```

Comment:
I don't think we need to use my branch for this , this can be merged directly to master with your transform version 2.1.9 and tested on https://jira.atypon.com/browse/LEAN-3477